www/src/components/email-capture-form-100days.js

```diff
@@ -0,0 +1,317 @@
+/** @jsx jsx */
+/** "formID is the only difference with email-capture-form.js  */
```

Comment:
If this is only difference, could we make passing `formID` as prop and reuse already existing component to avoid bloating our bundle?

Reply:
Yes, I agree. I got advice that this would be ok for now, and I can go back and refactor later

Reply:
We probably can ship as-is and iterate in follow up? What's the time sensivity of publishing this post?

Reply:
It needs to be up today. 

And I have a project in January to work on this form, and a few other things for Gatsby Marketing as part of me learning React.